src/bio/bio_context.c

```diff
@@ -275,6 +275,24 @@ bio_bs_hold(struct bio_blobstore *bbs)
 	return rc;
 }
 
+struct bio_xs_blobstore *
+bio_xs_blobstore_by_devid(struct bio_xs_context *xs_ctxt, uuid_t dev_uuid)
+{
+	enum smd_dev_type	 st;
+	struct bio_xs_blobstore *bxb = NULL;
+
+	for (st = SMD_DEV_TYPE_DATA; st < SMD_DEV_TYPE_MAX; st++) {
```

Comment:
Very minor comment but I would prefer using 0 as a starting point is it doesn't use implicit knowledge that DATA is the first entry in the enum.